packages/rainbowkit/src/wallets/walletConnectors/metaMask/metaMask.ts

```diff
@@ -10,7 +10,7 @@ export interface MetaMaskOptions {
   shimDisconnect?: boolean;
 }
 
-export function isMetaMask(ethereum: NonNullable<typeof window['ethereum']>) {
+function isMetaMask(ethereum: NonNullable<typeof window['ethereum']>) {
```

Comment:
FYI, this is out of sync with wagmi atm... We may need to either add [tocken pocket](https://github.com/wagmi-dev/wagmi/blob/main/packages/core/src/connectors/metaMask.ts#L122) or find a way to keep in sync with wagmi.

Reply:
Syncing up with wagmi is a good idea, but in the meantime I've updated our logic in this PR.